packages/fxa-settings/.license.header

```diff
@@ -0,0 +1,3 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Not sure about the location and name of this file, but gotta put it somewhere. 🤷

Reply:
I'll file a bug for potentially updating Payments to use the same approach, and we can centralize stuff there.